book/website/afterword/subprojects.md

```diff
@@ -17,11 +17,13 @@ These subprojects are often time-bound and are supported by community members wh
 ### Infrastructure Working Group Leads
 
 *Project Information: [GitHub issues](https://github.com/the-turing-way/the-turing-way/issues?q=is%3Aissue+is%3Aopen+label%3Ainfrastructure)*
+- Chair: [Danny Garside](https://the-turing-way.netlify.app/afterword/contributors-record#danny-garside)
+- Secretary: [Sarah Gibson](https://the-turing-way.netlify.app/afterword/contributors-record#sarah-gibson)
+- Member: [Brigitta Sipőcz](https://the-turing-way.netlify.app/afterword/contributors-record#brigitta-sipocz)
+- Member: [Jim Madge](https://the-turing-way.netlify.app/afterword/contributors-record#jim-madge)
+- Member: [Johana Bayer](https://the-turing-way.netlify.app/afterword/contributors-record#likeajumprope)
 
-- [Brigitta Sipőcz](https://the-turing-way.netlify.app/afterword/contributors-record#brigitta-sipocz)
-- [Danny Garside](https://the-turing-way.netlify.app/afterword/contributors-record#danny-garside)
-- [Jim Madge](https://the-turing-way.netlify.app/afterword/contributors-record#jim-madge)
-- [Sarah Gibson](https://the-turing-way.netlify.app/afterword/contributors-record#sarah-gibson)
+The Infrastructure working group chair rotates every 6 months. Next chair rotation allocated for September 2024.
```

Comment:
Is it worth saying who the group of people it rotates between is?
Is that what a co-chair is?

Reply:
I think it's worth adding it. What do you think now?

Reply:
What I remember from the meeting (and it would be worth checking 😅).

Danny and I were excited and _wanted_ to be chair.
Brigitta and Sarah were happy to be called but wouldn't ask to be chair.

So, the pool is Danny and Me?

Reading the meeting minutes again. I think co-chairs is more for when the responsibilities are being shared between multiple people simultaneously. So I think one chair is right :+1:.